### ISSUE TITLE
Parallelization

### ISSUE THREAD

**Issue by n1try** [skjolber/3d-bin-container-packing#114]
It would be nice to have a parallel version of the packagers. Would it be possible?

**Comment by skjolber** [comment]
Yes and no, some parts might be calculated in paralell, at least parts of the search for the smallest box.

**Comment by skjolber** [comment]
Also the brute force can be calculated in paralell, since each permutation is calculated by it self, having 100 permutations and 10 threads would mean 10 for each thread.

**Comment by n1try** [comment]
I'd appreciate a parallel version of at least the brute force packer! :)

**Comment by skjolber** [comment]
Puh, after a lot of searching I've adapted some python script for a permutator that handles BoxItems (i.e. multiple item of the same time) in the minimal way.

https://github.com/skjolber/3d-bin-container-packing/tree/multiThreading

So this would make it possible to process a predefined range of permutations in parallel. Each range would then be compared for the best result.

**Comment by skjolber** [comment]
This has been merged, but is pending release till I get me a JMH module.
